doc/developer-guides/hld/hv-partitionmode.rst

```diff
@@ -0,0 +1,358 @@
+.. _partition-mode-hld:
+
+Partition mode
+##############
+
+ACRN is type-1 hypervisor that supports running multiple guest operating
+systems (OS). Typically, the platform BIOS/boot-loader boots ACRN, and
+ACRN loads single or multiple guest OSes. Refer to :ref:`hv-startup` for
+details on the start-up flow of the ACRN hypervisor.
+
+ACRN supports two modes of operation: Sharing mode and Partition mode.
+This document describes ACRN's high-level design for Partition mode
+support.
+
+.. contents::
+   :depth: 2
+   :local:
+
+Introduction
+************
+
+ACRN, in partition mode, is used to provide guests with exclusive access
+to cores, memory, cache, and IO devices. There is no resource sharing
```

Comment:
Can we provide exclusive access to last level cache? If not, we may need to be more specific on 'exclusive access to cache' here.

Reply:
@junjiemao1 I added this as a placeholder before we add RDT support to ACRN. Since RDT support is in our roadmap, can we keep it here for now or remove it and add later when RDT implementation is completed?

Reply:
How about adding a note saying that the cache part is partitioned only after RDT is supported?